d2l-tspan.html

```diff
@@ -16,14 +16,34 @@
 				border: none;
 			}
 
-			#float {
+			.float,
+			.float-with-border {
 				position: absolute;
 				width: 100%;
-				border: var(--d2l-table-border);
-				border-bottom: none;
 				padding: 1rem;
 				left: 0;
-				box-sizing: border-box;
+				box-sizing: border-box;			
+			}
+
+			.float {
+				border: var(--d2l-table-border);
+				z-index: 0;
+				border-bottom: none;
+				box-shadow: 0 1px 0 var(--d2l-table-border-color);
+			}
+
+			:host([custom-border]) .float,
+			.float-with-border {
+				transition-property: border-color;
+				transition-timing-function: ease;
+				transition: border-color 0.5s, box-shadow 0.5s;
+			}
+
+			.float-with-border {
```

Comment:
I would rename this class to something like `.float-selected` instead of `.float-with-border` since the normal `.float` also has a border, just of a different colour and width